test/latest/Dockerfile

```
@@ -230,6 +230,9 @@ RUN install-tool skopeo 1.17.0
 # renovate: datasource=github-releases packageName=getsops/sops
 RUN install-tool sops v3.9.3
 
+# renovate: datasource=github-releases packageName=protocolbuffers/protobuf
+RUN install-tool protoc 29.2.0
```

```suggestion
RUN install-tool protoc 29.2
```
there is no such version